pkg/controllers/user/networkpolicy/netpol.go

```diff
@@ -292,6 +292,17 @@ func (npmgr *netpolMgr) handleHostNetwork() error {
 	}
 
 	systemNamespaces := GetSystemNamespaces()
```

Comment:
there is one more place in the code where we fetch system namespaces (line 95), so the fix should apply to both places